website/docs/cluster-management/profiles.mdx

```diff
@@ -78,6 +78,54 @@ spec:
 ...
 ```
 
+#### (Optional) Using a helm chart from a remote public/private repository
+The helm releases with the profiles can be added in a remote repository which can be referenced using a HelmRepository resource.The repository can be either public or private, although extra steps are required to use the private repo.
+
+In this example a public repo and branch is referenced directly where the helm releases are
+```yaml title="HelmRepository.yaml"
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: weaveworks-charts
+  namespace: flux-system
+spec:
+  interval: 1m
+  url: https://raw.githubusercontent.com/weaveworks/weave-gitops-profile-examples/gh-pages
```

Comment:
this url might be better as points to the webserver https://weaveworks.github.io/weave-gitops-profile-examples/